samples/v1/speech_transcribe_async.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// DO NOT EDIT! This is a generated sample ("LongRunningPromiseAwait",  "speech_transcribe_async")
+
+// sample-metadata:
+//   title: Transcribe Audio File using Long Running Operation (Local File) (LRO)
+//   description: Transcribe a long audio file using asynchronous speech recognition
+//   usage: node samples/v1/speech_transcribe_async.js [--local_file_path "resources/brooklyn_bridge.raw"]
+
+'use strict';
+
+// [START speech_transcribe_async]
+
+const {SpeechClient} = require('@google-cloud/speech').v1;
+
+const fs = require('fs');
+/**
+ * Transcribe a long audio file using asynchronous speech recognition
+ *
+ * @param localFilePath {string} Path to local audio file, e.g. /path/audio.wav
+ */
+async function sampleLongRunningRecognize(localFilePath) {
```

Comment:
Following the guidance here:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

We want to create an outer wrapping function called `main`, and then have the `async` function and import defined *inside* of it.  This is a great canonical sample:
https://github.com/googleapis/nodejs-bigquery/blob/master/samples/addColumnLoadAppend.js

Reply:
+1